gravityforms-multiple-form-instances.php

```diff
@@ -95,6 +97,14 @@ public function gform_get_form_filter( $form_string, $form ) {
 			'GFCalc(' . $form['id'] . ','                                       => 'GFCalc(' . $random_id . ',',
 			'gf_global["number_formats"][' . $form['id'] . ']'                  => 'gf_global["number_formats"][' . $random_id . ']',
 			'gform_next_button_' . $form['id'] . '_'                            => 'gform_next_button_' . $random_id . '_',
+			'multipart_params&quot;:{&quot;form_id&quot;'                       => 'multipart_params&quot;:{&quot;random_id&quot;:' . $random_id . ',&quot;form_id&quot;',
```

Comment:
Here we add the 'random_id' multipart_param, that is then simply accessible in the code, since plupload will have a reference to it in its 'settings.multipart_params.random_id' property.
The drawback is that it is also sent to the server while uploading files, and doesn't have any purpose in this context.